NAS2D/Resource/Font.cpp

```diff
@@ -268,11 +254,6 @@ namespace
 	}
 
 
-	/**
-	 * Generates a glyph map of all ASCII standard characters from 0 - 255.
-	 *
-	 * Internal function used to generate a glyph texture map from an TTF_Font struct.
-	 */
```

Comment:
The ASCII limitation may not be obvious. We should probably keep that comment.

Reply:
The datatype should probably be changed to `uint8_t` or `unsigned char`.

Reply:
What data type? The limitation comes from the fixed size used for the `std::vector`.